mongodbatlas/resource_mongodbatlas_database_user_test.go

```diff
@@ -167,7 +167,7 @@ func TestAccResourceMongoDBAtlasDatabaseUser_WithLabels(t *testing.T) {
 func TestAccResourceMongoDBAtlasDatabaseUser_importBasic(t *testing.T) {
 	projectID := os.Getenv("MONGODB_ATLAS_PROJECT_ID")
 
-	username := fmt.Sprintf("test_username_%s", acctest.RandString(5))
+	username := fmt.Sprintf("test-username-%s", acctest.RandString(5))
```

Comment:
Would be better if you use a function made for this case?, like this
```go
username := acctest.RandomWithPrefix("tf-acc-test-username")
```

Reply:
yes! thank you so much let me include it